### DISCUSSION TITLE
industry session

### DISCUSSION THREAD

**Discussion by ankitkushawaha1000** [github/india#216] [Students]
When i watched a industry session video then it shows complete and a dialog box which says error
and when i try to refresh the page then it shows incomplete.
why this is happening??

**Comment by dk162624** [comment]
again play  all indusrty session video together. and leave it

**Comment by dk162624** [comment]
your problem will be short out

**Comment by TheSuranaverse** [comment]
**FUTURE READY SEPARATE CHANNEL**

For smooth communication & discussion!
Link: https://discord.gg/ceMXzhfaka

Thank you 🙏
